### PR TITLE
Tests failing

### PR DESCRIPTION
**Pull request by felquis**
Investigation details are in the issue https://github.com/pagarme/react-event-components/issues/21

### CODE REVIEW

**Comment by felquis** [comment]
The only error now, is that window.requestAnimationFrame isn't defined!

Well it's better then before :dagger: we need to understand how Mocks works in Jest https://facebook.github.io/jest/docs/manual-mocks.html

File: src/react-event-components/Every.js [hunk 1]
```diff
@@ -6,14 +6,14 @@ module.exports = class Every extends Component {
     this.state = {
       intervalId: null,
       shouldTriggerNextFrame: true,
-      lastFrame: performance.now(),
+      lastFrame: Date.now(),
```
Comment (felquis) [inline src/react-event-components/Every.js]: I changed it just to avoid mocks, since nobody knows how to create then right now.

**Comment by felquis** [comment]
@grvcoelho could you help on this somehow?

**Comment by grvcoelho** [comment]
@felquis: Jest doesn't ship with requestAnimationFrame!

We have to polyfill it so it starts working.

A greate polyfill for this: https://gist.github.com/paulirish/1579671

**Comment by felquis** [comment]
In this [docs](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#initializing-test-environment) it says to use a `src/setupTests.js` file which worked for a simple mock, I created some tests but the coverage don't lie, theres tests to be written in the future.

screen shot 2016-12-12 at 1 34 05 am

**Comment by felquis** [comment]
The command "eval yarn" failed. Retrying, 2 of 3.

https://travis-ci.org/pagarme/react-event-components/jobs/183150461#L239

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - "node_modules"
+
+before_script:
+  - npm install -g yarn
+  - yarn
+
+script:
+  - npm test
+  - npm run build
+  - npm run demo-deploy
```
Comment (derekstavis) [inline .travis.yml]: We should remove this or any test that Travis runs will deploy to Github Pages automatically.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - "7.2.1"
+
+cache:
+  directities:
```
Comment (derekstavis) [inline .travis.yml]: Should be `directories`.

Comment (derekstavis) [inline .travis.yml]: You can also use [Yarn cache](https://docs.travis-ci.com/user/caching/#yarn-cache)

**Comment by derekstavis** [comment]
This is looking great! See #35 for fixing Travis error!

File: .travis.yml (hunk 2 above)
Comment (felquis) [inline .travis.yml]: Removed!

**Comment by felquis** [comment]
@derekstavis we did it! Lets right tests now \o/

**Comment by derekstavis** [comment]
Squash and merging! Awesome job here! :clap: :clap: :clap: :clap: :clap:

**Pull request merged**